src/main/java/gtPlusPlus/xmod/gregtech/loaders/RecipeGenExtruder.java

```diff
@@ -102,21 +98,19 @@ private void generateRecipes(final Material material) {
         }
 
         // Gear Recipe
-        if (ItemUtils.checkForInvalidItems(material.getIngot(1)) && ItemUtils.checkForInvalidItems(material.getGear(1)))
-            if (!material.isRadioactive) {
-                GTValues.RA.stdBuilder()
-                    .itemInputs(material.getIngot(4), shape_Gear)
-                    .itemOutputs(itemGear)
-                    .duration((int) Math.max(material.getMass() * 5L, 1))
-                    .eut(material.vVoltageMultiplier)
-                    .addTo(extruderRecipes);
+        if (null != (material.getIngot(1)) && null != (material.getGear(1))) if (!material.isRadioactive) {
```

Comment:
```suggestion
        if (material.getIngot(1) != null && material.getGear(1) != null && !material.isRadioactive) {
```
Same as the other, combines the if statements